services/app-api/forms/mcpar.json

```diff
@@ -1476,34 +1539,42 @@
                     "validation": "checkbox",
                     "props": {
                       "label": "C2.V.7 Monitoring methods",
-                      "hint": "What are the monitoring methods? Check all that apply or add free text if you select “Other”.",
+                      "hint": "Select the method(s) utilized to assess compliance with the state's 42 C.F.R. § 438.68 and 42 C.F.R. § 438.206 standards. If a method is not listed, select \"Custom Method\" and add details.",
                       "choices": [
                         {
                           "id": "gbWqxtPWaUC1yUbmhWA0UA",
                           "label": "Geomapping"
                         },
                         {
-                          "id": "G7GxcYWmhEqjrEQDBWRl3w",
-                          "label": "Plan provider roster review"
+                          "id": "8CMLyb311Nu2jAvjH2Shix",
+                          "label": "Plan Provider Directory Review"
+                        },
+                        {
+                          "id": "jV4HPfW1m52sQBPf626zYS",
+                          "label": "Secret Shopper: Network Participation"
                         },
                         {
-                          "id": "e14gKX61FkqBjIBeFDO7oA",
-                          "label": "Secret shopper calls"
+                          "id": "oPzVpoNQMKGjBZNv5rHwMx",
+                          "label": "Secret Shopper: Appointment Availability"
                         },
                         {
-                          "id": "cMhP8Jp6SUS9XvezaPLvLg",
-                          "label": "EVV data analysis"
+                          "id": "x7YAgoGbPfMRpZWcGSnjyf",
+                          "label": "Electronic Visit Verification Data Analysis"
                         },
                         {
                           "id": "bmQ1FmFPGk6S1g70ntAEcw",
-                          "label": "Review of grievances related to access"
+                          "label": "Review of Grievances Related to Access"
+                        },
+                        {
+                          "id": "r6FPy3varm5HbcBA44DjQ8",
+                          "label": "Encounter Data Analysis"
                         },
                         {
                           "id": "ozFC7ZVX7EWXZiNB2Ajj3g",
-                          "label": "Other, specify",
+                          "label": "Custom method",
                           "children": [
                             {
-                              "id": "accessMeasure_monitoringMethods-otherText",
+                              "id": "accessMeasure_monitoringMethods-customMethod",
```

Comment:
this value doesn't show up in the entity card when saved. If it were `otherText` our handling _might_ catch it? but yeah needs it in there somehow

Reply:
![Screenshot 2025-05-09 at 3 06 17 PM](https://github.com/user-attachments/assets/0fcd3c66-20d0-41fb-847d-f1e245d6e247)

Reply:
Good catch! Chatted with Kim and Anna about this and realized this isn't a new field, just a label change for the otherText field. That should sort this part out :D